src/ConnectedKubernetes/docs/Get-AzConnectedKubernetesUserCredential.md

```diff
@@ -227,7 +227,7 @@ COMPLEX PARAMETER PROPERTIES
 To create the parameters described below, construct a hash table containing the appropriate properties. For information on hash tables, run Get-Help about_Hash_Tables.
 
 
-`PROPERTY <IListClusterUserCredentialProperties>`: .
+PROPERTY <IListClusterUserCredentialProperties>: .
```

Comment:
I think this deleting is caused your autorest version. please run autorest --reset then re-run `autorest, ./build-module.ps1`    to upgrade autorest

Reply:
Thanks.
After "autorest --reset, autoreset, ./build-module.ps1" commands, the formats are recovered.